securedrop/journalist_app/admin.py

```diff
@@ -59,6 +59,7 @@ def update_submission_preferences():
         form = SubmissionPreferencesForm()
         if form.validate_on_submit():
             # The UI prompt ("prevent") is the opposite of the setting ("allow"):
+            flash(gettext("Preferences Saved."), "submission-preferences-success")
```

Comment:
Saved->saved (for consistency with other messages)

Reply:
2d95ebf77522b070d28e22b6db3d7f6f14165b26